examples/measure.html

```diff
@@ -1,10 +1,12 @@
 ---
 template: example.html
 title: Measure example
-shortdesc: Example of using the ol.interaction.Draw interaction for creating simple measuring application.
+shortdesc: Example of using the ol.interaction.Draw interaction for creating a simple measuring application.
 docs: >
-  <p><i>NOTE: If use geodesic measures is not checked, measure is done in simple way on projected plane. Earth curvature is not taken into account</i></p>
-tags: "draw, edit, measure, vector"
+  <p><i>NOTE: Area and length calculations are done using ol.proj.getLength and ol.proj.getArea to get answers in meters or square meters at any latitude.
+  The ol.geom.Linestring.getLength method does not give a length in meters for EPSG:3857 or EPSG:4326 linestrings.
```

Comment:
I have taken trouble to make the new methods return answers in projection units whenever possible. It is not possible to do so for projections with units of degrees. You can't convert a length in degrees calculated using Pythagoras  to metres. 

I am happy to drop the comment about th ol.geom.getLength and getArea from the example. 
Though this whole issue arose out of a desire to stop non GIS experts making mistakes using those methods improperly. 

METERS_PER_UNIT is not meaningful for Pixel Projections.